features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioPalGap.cpp

```diff
@@ -398,32 +399,197 @@ void Gap::gap_handler(const wsfMsgHdr_t* msg) {
     }
 
     connection_handle_t handle = (connection_handle_t)msg->param;
+    EventHandler *handler = get_gap()._pal_event_handler;
+
 
     switch(msg->event) {
-        case DM_PHY_READ_IND:
-            if (get_gap()._pal_event_handler) {
-                const hciLeReadPhyCmdCmplEvt_t* evt = (const hciLeReadPhyCmdCmplEvt_t*)msg;
-
-                get_gap()._pal_event_handler->on_read_phy(
-                    (hci_error_code_t::type)msg->status,
-                    handle,
-                    (ble::phy_t::type)evt->txPhy,
-                    (ble::phy_t::type)evt->rxPhy
-                );
+        case DM_PHY_READ_IND: {
+            if (!handler) {
+                break;
+            }
+            const hciLeReadPhyCmdCmplEvt_t* evt = (const hciLeReadPhyCmdCmplEvt_t*)msg;
+
+            handler->on_read_phy(
+                (hci_error_code_t::type)msg->status,
+                handle,
+                (ble::phy_t::type)evt->txPhy,
+                (ble::phy_t::type)evt->rxPhy
+            );
+        } break;
```

Comment:
Unless it is the style already adopted throughout this file, then pretty sure 'break' should be on it's own line and shouldn't it be inside the { } ? Though again it should be checked against the coding standards as to whether { } should be being used with case statements anyway.

Reply:
How would you declare your local case variables without opening a scope ? 
I can place break statement one line bellow `}`.

Reply:
I fixed style of this file too.